src/harness/fake_sas.py

```diff
@@ -222,6 +237,18 @@ def do_POST(self):
     self.end_headers()
     self.wfile.write(json.dumps(response))
 
+  def do_GET(self):
+    """Handles GET requests."""
+    path, value = self._parseUrl(self.path)
+    if path == "/v1.0/sas_impl":
```

Comment:
The HTTP GET and POST methods shall be used for all SAS-SAS requests. The URL endpoints for SAS-SAS requests are described in Table 2. Requests are sent to the SAS URL path which includes the string “/{sas_version_number}/{sas_record_type}”...